lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -314,12 +314,22 @@ void LocalEnforcer::execute_actions(
         handle_activate_service_action(session_map, action_p, session_update);
         break;
       case REDIRECT:
-        // This is GY based REDIRECT, GX redirect will come in as a regular rule
-        start_redirect_flow_install(session_map, action_p, session_update);
-        break;
       case RESTRICT_ACCESS:
-        MLOG(MWARNING) << "RESTRICT_ACCESS mode is unsupported"
-                       << ", will just terminate the service.";
+        {
+          // Bundle up all info into this struct so that we don't have to pass around
```

Comment:
Maybe we can make a function attached to `ServiceAction` that spits out the `FinalActionInstallInfo`. This way the REDIRECT/RESTRICT cases are short and sweet.

Reply:
FinalActionInstallInfo definition is private within LocalEnforcer and there is no sense on moving it outside to ServiceAction as It is only needed by LE.
Instead, I added a new function populate_final_action_install_info which provides the same result.